templating/template.go

```diff
@@ -42,6 +44,31 @@ func Template(environment map[string]interface{}, template v1.Template) (string,
 				return "", errors.Wrapf(err, "error setting %s", k)
 			}
 		}
+		err := vm.Set("findConfigItem", func(call otto.FunctionCall) otto.Value {
+			configType, _ := call.Argument(0).ToString()
+			configName, _ := call.Argument(1).ToString()
+			configItemParams := pkg.Config{
+				ExternalType: configType,
+				Name:         configName,
+			}
+			configItem, err := db.FindConfig(configItemParams)
+			if err != nil {
+				logger.Errorf("Error fetching config item for js: %v", err)
+				emptyObj, _ := vm.ToValue(map[string]string{})
+				return emptyObj
+			}
+
+			configObject := map[string]interface{}{
+				"configs":    *configItem.Spec,
```

Comment:
Why can't we just return the configItem  directly ?

Reply:
@moshloop In the issue description, you mentioned we should expose keys `properties` and `configs`